tests/blob-api.js

```diff
@@ -1,124 +1,106 @@
-
-  t.ok(await timeoutException(getPort(server.server)))
-
-  await new Promise((resolve) => {
-    server.listen({ port: 3456 }, (err, address) => {
-      resolve(address)
-    })
-  })
-
-  const port = await getPort(server.server)
-
-  t.is(typeof port, 'number')
-  t.is(port, 3456)
-
-  t.teardown(async () => {
-    await server.close()
-  })
-})
-
-test('get port after listening event with unset port', async (t) => {
-  const { blobStore } = createBlobStore()
-  const server = await createBlobServer({ blobStore })
-
-  t.ok(await timeoutException(getPort(server.server)))
-
-  await new Promise((resolve) => {
-    server.listen({ port: 0 }, (err, address) => {
-      resolve(address)
-    })
-  })
-
-  const port = await getPort(server.server)
-
-  t.is(typeof port, 'number', 'port is a number')
-  t.teardown(async () => {
-    await server.close()
-  })
-})
-
-test('get url from blobId', async (t) => {
-  const projectId = '1234'
-  const type = 'photo'
-  const variant = 'original'
-  const name = '1234'
-
-  const { blobStore } = createBlobStore()
-  const blobServer = await createBlobServer({ blobStore })
-  const blobApi = new BlobApi({ projectId: '1234', blobStore, blobServer })
-
-  await new Promise((resolve) => {
-    blobServer.listen({ port: 0 }, (err, address) => {
-      resolve(address)
-    })
-  })
-
-  const url = await blobApi.getUrl({
-    driveId: blobStore.writerDriveId,
-    type,
-    variant,
-    name,
-  })
-
-  t.is(
-    url,
-    `http://127.0.0.1:${blobServer.server.address().port}/${projectId}/${
-      blobStore.writerDriveId
-    }/${type}/${variant}/${name}`
-  )
-  t.teardown(async () => {
-    await blobServer.close()
-  })
-})
 
 test('create blobs', async (t) => {
   const { blobStore } = createBlobStore()
-  const blobServer = createBlobServer({ blobStore })
-  const blobApi = new BlobApi({ projectId: '1234', blobStore, blobServer })
 
-  await new Promise((resolve) => {
-    blobServer.listen({ port: 0 }, (err, address) => {
-      resolve(address)
-    })
+  const blobApi = new BlobApi({
+    projectId: randomBytes(32).toString('hex'),
+    blobStore,
+    getMediaBaseUrl: async () => 'http://127.0.0.1:8080/blobs',
   })
 
   const directory = fileURLToPath(
     new URL('./fixtures/blob-api/', import.meta.url)
   )
 
   const hash = createHash('sha256')
-  const content = await fs.readFile(join(directory, 'original.png'))
-  hash.update(content)
+  const originalContent = await fs.readFile(join(directory, 'original.png'))
+  hash.update(originalContent)
 
   const attachment = await blobApi.create(
     {
       original: join(directory, 'original.png'),
       preview: join(directory, 'preview.png'),
       thumbnail: join(directory, 'thumbnail.png'),
     },
-    {
-      mimeType: 'image/png',
-    }
+    { mimeType: 'image/png' }
   )
 
   t.is(attachment.driveId, blobStore.writerDriveId)
   t.is(attachment.type, 'photo')
   t.alike(attachment.hash, hash.digest('hex'))
```

Comment:
```suggestion
//  t.alike(attachment.hash, hash.digest('hex'))
```

maybe just comment this out and address in a follow-up?